{"code":"23505","constraint":"users_email_key","detail":"Ya existe la llave (email)=(user200@example.com).","file":"nbtinsert.c","length":213,"level":"error","line":"673","message":"Error creating user: llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»","name":"error","routine":"_bt_check_unique","schema":"public","severity":"ERROR","stack":"error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async createUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:51:22)","table":"users","timestamp":"2024-10-28T06:54:13.635Z"}
{
  "timestamp": "2024-10-28T07:05:09.722Z",
  "level": "error",
  "message": "Error creating user: llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 213,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(user200@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async createUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:51:22)"
}
{
  "timestamp": "2024-10-28T07:05:31.976Z",
  "level": "error",
  "message": "Error creating user: llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 213,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(user200@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async createUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:51:22)"
}
{
  "timestamp": "2024-10-28T07:05:32.658Z",
  "level": "error",
  "message": "Error creating user: llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 213,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(user200@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async createUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:51:22)"
}
{
  "timestamp": "2024-10-28T07:06:49.828Z",
  "level": "error",
  "message": "Error creating user: llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 213,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(user200@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async createUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:51:22)"
}
{
  "timestamp": "2024-10-28T07:12:02.779Z",
  "level": "error",
  "message": "Error getting all users: result is not defined result is not defined",
  "stack": "ReferenceError: result is not defined\n    at getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:10:31)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
}
{
  "timestamp": "2024-10-28T07:12:23.176Z",
  "level": "error",
  "message": "Error getting all users: result is not defined result is not defined",
  "stack": "ReferenceError: result is not defined\n    at getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:10:31)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
}
{
  "timestamp": "2024-10-28T07:12:45.173Z",
  "level": "error",
  "message": "Error getting all users: result is not defined result is not defined",
  "stack": "ReferenceError: result is not defined\n    at getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:10:31)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
}
{
  "timestamp": "2024-10-28T07:13:16.584Z",
  "level": "error",
  "message": "Error getting all users: result is not defined result is not defined",
  "stack": "ReferenceError: result is not defined\n    at getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:10:31)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
}
{
  "timestamp": "2024-11-01T05:09:53.991Z",
  "level": "error",
  "message": "Error getting all users: Port should be >= 0 and < 65536. Received type number (NaN). Port should be >= 0 and < 65536. Received type number (NaN).",
  "code": "ERR_SOCKET_BAD_PORT",
  "stack": "RangeError [ERR_SOCKET_BAD_PORT]: Port should be >= 0 and < 65536. Received type number (NaN).\n    at validatePort (node:internal/validators:424:11)\n    at lookupAndConnect (node:net:1289:5)\n    at Socket.connect (node:net:1246:5)\n    at Connection.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\connection.js:43:17)\n    at Client._connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:112:11)\n    at Client.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:161:12)\n    at BoundPool.newClient (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:237:12)\n    at BoundPool.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:212:10)\n    at BoundPool.query (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:399:10)\n    at getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:6:33)"
}
{
  "timestamp": "2024-11-01T05:11:05.848Z",
  "level": "error",
  "message": "Error getting all users: Port should be >= 0 and < 65536. Received type number (NaN). Port should be >= 0 and < 65536. Received type number (NaN).",
  "code": "ERR_SOCKET_BAD_PORT",
  "stack": "RangeError [ERR_SOCKET_BAD_PORT]: Port should be >= 0 and < 65536. Received type number (NaN).\n    at validatePort (node:internal/validators:424:11)\n    at lookupAndConnect (node:net:1289:5)\n    at Socket.connect (node:net:1246:5)\n    at Connection.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\connection.js:43:17)\n    at Client._connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:112:11)\n    at Client.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:161:12)\n    at BoundPool.newClient (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:237:12)\n    at BoundPool.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:212:10)\n    at BoundPool.query (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:399:10)\n    at getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:6:33)"
}
{
  "timestamp": "2024-11-01T05:11:52.133Z",
  "level": "error",
  "message": "Error getting all users: Port should be >= 0 and < 65536. Received type number (NaN). Port should be >= 0 and < 65536. Received type number (NaN).",
  "code": "ERR_SOCKET_BAD_PORT",
  "stack": "RangeError [ERR_SOCKET_BAD_PORT]: Port should be >= 0 and < 65536. Received type number (NaN).\n    at validatePort (node:internal/validators:424:11)\n    at lookupAndConnect (node:net:1289:5)\n    at Socket.connect (node:net:1246:5)\n    at Connection.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\connection.js:43:17)\n    at Client._connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:112:11)\n    at Client.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:161:12)\n    at BoundPool.newClient (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:237:12)\n    at BoundPool.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:212:10)\n    at BoundPool.query (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:399:10)\n    at getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:6:33)"
}
{
  "timestamp": "2024-11-01T05:12:00.102Z",
  "level": "error",
  "message": "Error getting all users: Port should be >= 0 and < 65536. Received type number (NaN). Port should be >= 0 and < 65536. Received type number (NaN).",
  "code": "ERR_SOCKET_BAD_PORT",
  "stack": "RangeError [ERR_SOCKET_BAD_PORT]: Port should be >= 0 and < 65536. Received type number (NaN).\n    at validatePort (node:internal/validators:424:11)\n    at lookupAndConnect (node:net:1289:5)\n    at Socket.connect (node:net:1246:5)\n    at Connection.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\connection.js:43:17)\n    at Client._connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:112:11)\n    at Client.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:161:12)\n    at BoundPool.newClient (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:237:12)\n    at BoundPool.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:212:10)\n    at BoundPool.query (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:399:10)\n    at getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:6:33)"
}
{
  "timestamp": "2024-11-01T05:12:19.681Z",
  "level": "error",
  "message": "Error getting all users: Port should be >= 0 and < 65536. Received type number (NaN). Port should be >= 0 and < 65536. Received type number (NaN).",
  "code": "ERR_SOCKET_BAD_PORT",
  "stack": "RangeError [ERR_SOCKET_BAD_PORT]: Port should be >= 0 and < 65536. Received type number (NaN).\n    at validatePort (node:internal/validators:424:11)\n    at lookupAndConnect (node:net:1289:5)\n    at Socket.connect (node:net:1246:5)\n    at Connection.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\connection.js:43:17)\n    at Client._connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:112:11)\n    at Client.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:161:12)\n    at BoundPool.newClient (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:237:12)\n    at BoundPool.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:212:10)\n    at BoundPool.query (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:399:10)\n    at getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:6:33)"
}
{
  "timestamp": "2024-11-01T05:12:20.254Z",
  "level": "error",
  "message": "Error getting all users: Port should be >= 0 and < 65536. Received type number (NaN). Port should be >= 0 and < 65536. Received type number (NaN).",
  "code": "ERR_SOCKET_BAD_PORT",
  "stack": "RangeError [ERR_SOCKET_BAD_PORT]: Port should be >= 0 and < 65536. Received type number (NaN).\n    at validatePort (node:internal/validators:424:11)\n    at lookupAndConnect (node:net:1289:5)\n    at Socket.connect (node:net:1246:5)\n    at Connection.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\connection.js:43:17)\n    at Client._connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:112:11)\n    at Client.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:161:12)\n    at BoundPool.newClient (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:237:12)\n    at BoundPool.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:212:10)\n    at BoundPool.query (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:399:10)\n    at getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:6:33)"
}
{
  "timestamp": "2024-11-01T05:12:20.468Z",
  "level": "error",
  "message": "Error getting all users: Port should be >= 0 and < 65536. Received type number (NaN). Port should be >= 0 and < 65536. Received type number (NaN).",
  "code": "ERR_SOCKET_BAD_PORT",
  "stack": "RangeError [ERR_SOCKET_BAD_PORT]: Port should be >= 0 and < 65536. Received type number (NaN).\n    at validatePort (node:internal/validators:424:11)\n    at lookupAndConnect (node:net:1289:5)\n    at Socket.connect (node:net:1246:5)\n    at Connection.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\connection.js:43:17)\n    at Client._connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:112:11)\n    at Client.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:161:12)\n    at BoundPool.newClient (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:237:12)\n    at BoundPool.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:212:10)\n    at BoundPool.query (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:399:10)\n    at getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:6:33)"
}
{
  "timestamp": "2024-11-01T05:12:20.702Z",
  "level": "error",
  "message": "Error getting all users: Port should be >= 0 and < 65536. Received type number (NaN). Port should be >= 0 and < 65536. Received type number (NaN).",
  "code": "ERR_SOCKET_BAD_PORT",
  "stack": "RangeError [ERR_SOCKET_BAD_PORT]: Port should be >= 0 and < 65536. Received type number (NaN).\n    at validatePort (node:internal/validators:424:11)\n    at lookupAndConnect (node:net:1289:5)\n    at Socket.connect (node:net:1246:5)\n    at Connection.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\connection.js:43:17)\n    at Client._connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:112:11)\n    at Client.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:161:12)\n    at BoundPool.newClient (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:237:12)\n    at BoundPool.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:212:10)\n    at BoundPool.query (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:399:10)\n    at getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:6:33)"
}
{
  "timestamp": "2024-11-01T05:12:21.024Z",
  "level": "error",
  "message": "Error getting all users: Port should be >= 0 and < 65536. Received type number (NaN). Port should be >= 0 and < 65536. Received type number (NaN).",
  "code": "ERR_SOCKET_BAD_PORT",
  "stack": "RangeError [ERR_SOCKET_BAD_PORT]: Port should be >= 0 and < 65536. Received type number (NaN).\n    at validatePort (node:internal/validators:424:11)\n    at lookupAndConnect (node:net:1289:5)\n    at Socket.connect (node:net:1246:5)\n    at Connection.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\connection.js:43:17)\n    at Client._connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:112:11)\n    at Client.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg\\lib\\client.js:161:12)\n    at BoundPool.newClient (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:237:12)\n    at BoundPool.connect (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:212:10)\n    at BoundPool.query (D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:399:10)\n    at getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:6:33)"
}
{
  "timestamp": "2024-11-01T05:18:44.043Z",
  "level": "error",
  "message": "Error getting all users: getaddrinfo ENOTFOUND \"localhost\"; getaddrinfo ENOTFOUND \"localhost\";",
  "errno": -3008,
  "code": "ENOTFOUND",
  "syscall": "getaddrinfo",
  "hostname": "\"localhost\";",
  "stack": "Error: getaddrinfo ENOTFOUND \"localhost\";\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:6:22)"
}
{
  "timestamp": "2024-11-01T05:19:06.574Z",
  "level": "error",
  "message": "Error getting user 1: getaddrinfo ENOTFOUND \"localhost\"; getaddrinfo ENOTFOUND \"localhost\";",
  "errno": -3008,
  "code": "ENOTFOUND",
  "syscall": "getaddrinfo",
  "hostname": "\"localhost\";",
  "stack": "Error: getaddrinfo ENOTFOUND \"localhost\";\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getUserById (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:21:22)"
}
{
  "timestamp": "2024-11-01T05:20:31.786Z",
  "level": "error",
  "message": "Error getting user 1: getaddrinfo ENOTFOUND \"localhost\"; getaddrinfo ENOTFOUND \"localhost\";",
  "errno": -3008,
  "code": "ENOTFOUND",
  "syscall": "getaddrinfo",
  "hostname": "\"localhost\";",
  "stack": "Error: getaddrinfo ENOTFOUND \"localhost\";\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getUserById (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:21:22)"
}
{
  "timestamp": "2024-11-01T05:20:39.420Z",
  "level": "error",
  "message": "Error getting all users: getaddrinfo ENOTFOUND \"localhost\"; getaddrinfo ENOTFOUND \"localhost\";",
  "errno": -3008,
  "code": "ENOTFOUND",
  "syscall": "getaddrinfo",
  "hostname": "\"localhost\";",
  "stack": "Error: getaddrinfo ENOTFOUND \"localhost\";\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:6:22)"
}
{
  "timestamp": "2024-11-02T23:19:20.788Z",
  "level": "error",
  "message": "Error creating user: llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 213,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(user204@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async createUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:53:22)"
}
{
  "timestamp": "2024-11-03T07:26:08.796Z",
  "level": "error",
  "message": "Error getting all users: rows is not defined rows is not defined",
  "stack": "ReferenceError: rows is not defined\n    at getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:12:31)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
}
{
  "timestamp": "2024-11-03T07:26:40.235Z",
  "level": "error",
  "message": "Error getting all users: rows is not defined rows is not defined",
  "stack": "ReferenceError: rows is not defined\n    at getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:12:31)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
}
{
  "timestamp": "2024-11-03T07:32:57.038Z",
  "level": "error",
  "message": "Error getting user 1: rows is not defined rows is not defined",
  "stack": "ReferenceError: rows is not defined\n    at getUserById (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:32:26)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
}
{
  "timestamp": "2024-11-03T07:43:52.736Z",
  "level": "error",
  "message": "Error creating user: INSERT tiene más expresiones que columnas de destino INSERT tiene más expresiones que columnas de destino",
  "length": 122,
  "name": "error",
  "severity": "ERROR",
  "code": "42601",
  "position": "49",
  "file": "analyze.c",
  "line": "1031",
  "routine": "transformInsertRow",
  "stack": "error: INSERT tiene más expresiones que columnas de destino\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async createUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:56:22)"
}
{
  "timestamp": "2024-11-03T07:45:45.929Z",
  "level": "error",
  "message": "Error creating user: llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 214,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async createUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:56:22)"
}
{
  "timestamp": "2024-11-05T05:09:57.315Z",
  "level": "error",
  "message": "Error getting all users  error de sintaxis en o cerca de «ASCe» error de sintaxis en o cerca de «ASCe»",
  "length": 103,
  "name": "error",
  "severity": "ERROR",
  "code": "42601",
  "position": "33",
  "file": "scan.l",
  "line": "1245",
  "routine": "scanner_yyerror",
  "stack": "error: error de sintaxis en o cerca de «ASCe»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/models/user.model.js:4:20)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:14:19)"
}
{
  "timestamp": "2024-11-05T05:10:08.174Z",
  "level": "error",
  "message": "Error getting all users  error de sintaxis en o cerca de «ASCe» error de sintaxis en o cerca de «ASCe»",
  "length": 103,
  "name": "error",
  "severity": "ERROR",
  "code": "42601",
  "position": "33",
  "file": "scan.l",
  "line": "1245",
  "routine": "scanner_yyerror",
  "stack": "error: error de sintaxis en o cerca de «ASCe»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/models/user.model.js:4:20)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:14:19)"
}
{
  "timestamp": "2024-11-05T05:17:00.127Z",
  "level": "error",
  "message": "error de sintaxis en o cerca de «ASCe» error de sintaxis en o cerca de «ASCe»",
  "length": 103,
  "name": "error",
  "severity": "ERROR",
  "code": "42601",
  "position": "33",
  "file": "scan.l",
  "line": "1245",
  "routine": "scanner_yyerror",
  "stack": "error: error de sintaxis en o cerca de «ASCe»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/models/user.model.js:4:20)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:14:19)"
}
{
  "timestamp": "2024-11-05T05:41:03.867Z",
  "level": "error",
  "message": "error de sintaxis en o cerca de «ASCe» error de sintaxis en o cerca de «ASCe»",
  "length": 103,
  "name": "error",
  "severity": "ERROR",
  "code": "42601",
  "position": "33",
  "file": "scan.l",
  "line": "1245",
  "routine": "scanner_yyerror",
  "stack": "error: error de sintaxis en o cerca de «ASCe»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/models/user.model.js:4:20)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:14:19)"
}
{
  "timestamp": "2024-11-05T05:46:37.733Z",
  "level": "error",
  "message": "error de sintaxis en o cerca de «ASCe» error de sintaxis en o cerca de «ASCe»",
  "length": 103,
  "name": "error",
  "severity": "ERROR",
  "code": "42601",
  "position": "33",
  "file": "scan.l",
  "line": "1245",
  "routine": "scanner_yyerror",
  "stack": "error: error de sintaxis en o cerca de «ASCe»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/models/user.model.js:4:20)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:14:19)"
}
{
  "timestamp": "2024-11-05T05:47:05.995Z",
  "level": "error",
  "message": "no se pudo determinar el tipo del parámetro $1 no se pudo determinar el tipo del parámetro $1",
  "length": 126,
  "name": "error",
  "severity": "ERROR",
  "code": "42P18",
  "file": "postgres.c",
  "line": "741",
  "routine": "pg_analyze_and_rewrite_varparams",
  "stack": "error: no se pudo determinar el tipo del parámetro $1\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getUserById (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/models/user.model.js:9:20)\n    at async getUserById (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:30:18)"
}
{
  "timestamp": "2024-11-05T05:51:10.438Z",
  "level": "error",
  "message": "llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 215,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe2@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async createUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/models/user.model.js:19:20)\n    at async createUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:76:18)"
}
{
  "timestamp": "2024-11-05T06:34:24.635Z",
  "level": "error",
  "message": "Cannot access 'email' before initialization Cannot access 'email' before initialization",
  "stack": "ReferenceError: Cannot access 'email' before initialization\n    at createUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:74:49)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
}
{
  "timestamp": "2024-11-05T06:35:13.705Z",
  "level": "error",
  "message": "Cannot access 'email' before initialization Cannot access 'email' before initialization",
  "stack": "ReferenceError: Cannot access 'email' before initialization\n    at createUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:74:49)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
}
{
  "timestamp": "2024-11-05T06:36:30.678Z",
  "level": "error",
  "message": "Cannot access 'email' before initialization Cannot access 'email' before initialization",
  "stack": "ReferenceError: Cannot access 'email' before initialization\n    at createUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:74:49)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
}
{
  "timestamp": "2024-11-05T06:36:54.697Z",
  "level": "error",
  "message": "Cannot access 'email' before initialization Cannot access 'email' before initialization",
  "stack": "ReferenceError: Cannot access 'email' before initialization\n    at createUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:79:47)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
}
{
  "timestamp": "2024-11-05T18:36:03.139Z",
  "level": "error",
  "message": "llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 215,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe2@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:106:22)"
}
{
  "timestamp": "2024-11-05T18:51:52.307Z",
  "level": "error",
  "message": "llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 215,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe2@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:106:22)"
}
{
  "timestamp": "2024-11-05T18:51:54.414Z",
  "level": "error",
  "message": "llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 215,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe2@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:106:22)"
}
{
  "timestamp": "2024-11-05T20:00:24.862Z",
  "level": "error",
  "message": "llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 215,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe2@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:115:22)"
}
{
  "timestamp": "2024-11-05T20:00:46.643Z",
  "level": "error",
  "message": "llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 215,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe2@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:115:22)"
}
{
  "timestamp": "2024-11-05T20:01:11.981Z",
  "level": "error",
  "message": "llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 215,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe2@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:115:22)"
}
{
  "timestamp": "2024-11-05T21:26:17.193Z",
  "level": "error",
  "message": "llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 215,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe2@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:115:22)"
}
{
  "timestamp": "2024-11-05T21:39:44.462Z",
  "level": "error",
  "message": "llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 215,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe2@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:115:22)"
}
{
  "timestamp": "2024-11-05T21:39:55.358Z",
  "level": "error",
  "message": "llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 215,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe2@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:115:22)"
}
{
  "timestamp": "2024-11-05T21:40:47.720Z",
  "level": "error",
  "message": "llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 215,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe2@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:116:22)"
}
{
  "timestamp": "2024-11-05T21:41:36.796Z",
  "level": "error",
  "message": "llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 215,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe2@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:116:22)"
}
{
  "timestamp": "2024-11-05T21:47:53.527Z",
  "level": "error",
  "message": "llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 215,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe2@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:116:22)"
}
{
  "timestamp": "2024-11-05T21:48:05.954Z",
  "level": "error",
  "message": "llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 215,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe2@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:116:22)"
}
{
  "timestamp": "2024-11-05T21:48:20.634Z",
  "level": "error",
  "message": "llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 215,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe2@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:116:22)"
}
{
  "timestamp": "2024-11-05T21:48:31.232Z",
  "level": "error",
  "message": "llave duplicada viola restricción de unicidad «users_email_key» llave duplicada viola restricción de unicidad «users_email_key»",
  "length": 215,
  "name": "error",
  "severity": "ERROR",
  "code": "23505",
  "detail": "Ya existe la llave (email)=(john.doe2@example.com).",
  "schema": "public",
  "table": "users",
  "constraint": "users_email_key",
  "file": "nbtinsert.c",
  "line": "673",
  "routine": "_bt_check_unique",
  "stack": "error: llave duplicada viola restricción de unicidad «users_email_key»\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async updateUser (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:116:22)"
}
{
  "timestamp": "2024-11-23T18:49:29.703Z",
  "level": "error",
  "message": "",
  "code": "ECONNREFUSED",
  "stack": "AggregateError\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/models/user.model.js:4:20)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:14:19)"
}
{
  "timestamp": "2024-11-23T18:49:31.043Z",
  "level": "error",
  "message": "",
  "code": "ECONNREFUSED",
  "stack": "AggregateError\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/models/user.model.js:4:20)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:14:19)"
}
{
  "timestamp": "2024-11-23T18:58:32.727Z",
  "level": "error",
  "message": "getaddrinfo ENOTFOUND localhost2 getaddrinfo ENOTFOUND localhost2",
  "errno": -3008,
  "code": "ENOTFOUND",
  "syscall": "getaddrinfo",
  "hostname": "localhost2",
  "stack": "Error: getaddrinfo ENOTFOUND localhost2\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/models/user.model.js:4:20)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:14:19)"
}
{
  "timestamp": "2024-11-23T18:59:30.185Z",
  "level": "error",
  "message": "Connection terminated due to connection timeout Connection terminated due to connection timeout",
  "stack": "Error: Connection terminated due to connection timeout\n    at D:\\Personal\\Tecnologias\\Node\\postgres-node-apirest\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/models/user.model.js:4:20)\n    at async getAllUsers (file:///D:/Personal/Tecnologias/Node/postgres-node-apirest/src/controllers/user.controllers.js:14:19)"
}
